src/Login.css

```diff
@@ -0,0 +1,103 @@
+body, html {
+    margin: 0;
+    padding: 0;
+    overflow: hidden; 
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+      'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+      sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+  }
+  
+  .login-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100vw; 
+    height: 100vh; 
+    background-image: url('./Assets/Band2.png');
+    background-size: cover;
+    background-position: center;
+  }
+
+  @media (max-width: 767px) {
+    .login-container {
+      background-image: url('./Assets/Band2.png');
+    }
+  }
+  @media (max-width: 767px) {
+    body, html {
+      overflow: auto;
+    }
+}
+
+.logo-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 20px;
+  background-color: rgba(134, 93, 255, 0.377);
+  border-radius: 8px;
+  box-shadow: 0px 4px 10px rgba(0, 0, 0, 0.1);
+}
+
+.logo {
+  width: 400px;
+  height: 330px;
+}
+
+
+.username-field,
+.password-field {
+  padding: 8px;
+  margin-right: 10px;
+}
+
+.login-button {
+  background-color: #865DFF;
+  color: white;
+  border: none;
+  padding: 8px 20px;
+  cursor: pointer;
+}
+
+.login-button:hover {
+  background-color: #6546C4;
+}
+
+@media (min-width: 412px) and (max-width: 915px) {
+    .logo {
+      width: 300px;
+      height: 247.5px;
+    }
```

Comment:
This breaks at 411px where the width of the .logo reverts back to the original. I would suggest not putting a range on the width and just properly order them so that some of the styling still applies to screens smaller than 412px

Reply:
implementing.!

Reply:
cleaned up all micro adjustments. should be all flex container based